modules/candidate_parameters/README.md

```diff
@@ -0,0 +1,94 @@
+
+ - useProband 
+    - This setting displays/hides the Proband tab of the module.
+
+ - useFamilyID
+    - This setting displays/hides the Family tab of the module.
+
+ - useConsent 
+    - This setting displays/hides the Consent tab of the module.
+ 
+ #### Additional Parameters
+ In addition to the Settings above, it is possible to add candidate parameters to the Candidate Info page. To do so, values must be added in SQL following the example below. These queries will add a single additonal dropdown with label _"Visit plan for candidate"_, field name _"candidate_plan"_, and options _"6month"_, _"12month"_. When candidate information is entered for this field, the value is saved in the `parameter_candidate` table in SQL.
+ 
+ Example:
+ ```
+ INSERT INTO `parameter_type_category` (Name, Type) VALUES ('Candidate Parameters','Metavars');
+ INSERT INTO parameter_type (Name, Type, Description, SourceFrom, Queryable) VALUES 
+  ('candidate_plan', "enum('6month', '12month')", 'Visit plan for candidate', 'parameter_candidate', 1);
+ 
+  INSERT INTO parameter_type_category_rel (ParameterTypeID, ParameterTypeCategoryID) SELECT 
+  pt.ParameterTypeID, ptc.ParameterTypeCategoryID FROM parameter_type_category ptc, 
+  parameter_type pt WHERE ptc.Name='Candidate Parameters' AND pt.Name='candidate_plan';
+ ```
+ 
+ #### Additional Consents
+ It is possible to have multiple consent types in the Consent Status tab. In order to add consents, first make sure the useConsent option is set to `true`, second fillow the instructions below.
+ 
+ 1. Populate the config file with additional Consent tagsets. For example, for consent to draw blood:
+
+         ```xml
+         <ConsentModule>
+             <useConsent>true</useConsent>
+             <Consent>
+                 <name>study_consent</name>
+                 <label>Consent to Study</label>
+             </Consent>
+             <Consent>
+                 <name>draw_blood_consent</name>
+                 <label>Consent to draw blood</label>
+             </Consent>
+         </ConsentModule>
+         ```
+ 2. Then, alter both participant_status and consent_info_history table schemas, adding 3 columns for the new consent type, to store consent status, date and withdrawal status. For example:
+
+         ```sql
+         ALTER TABLE participant_status ADD COLUMN draw_blood_consent enum('yes','no','not_answered');
+         ALTER TABLE participant_status ADD COLUMN draw_blood_consent_date date;
+         ALTER TABLE participant_status ADD COLUMN draw_blood_consent_withdrawal date;
+         ALTER TABLE consent_info_history ADD COLUMN draw_blood_consent enum('yes','no','not_answered');
+         ALTER TABLE consent_info_history ADD COLUMN draw_blood_consent_date date;
+         ALTER TABLE consent_info_history ADD COLUMN draw_blood_consent_withdrawal date;
+         ```
+
+## Interactions with LORIS
+
+- Information collected in this modules is exported towards the Data Query Tool (DQT) 
```

Comment:
"Information collected in this module can be exported to the Data Query Tool (DQT) by using the `/tools/CouchDB_Import_Demographics.php` script."

Reply:
partially done